tests/test_dss.py

```diff
@@ -245,13 +245,10 @@ def test_to_json(self, web3: Web3, mcd: DssDeployment):
         assert "MCD_DAI" in dict
         assert len(dict) > 20
 
-    def test_from_network(self, web3: Web3):
```

Comment:
This is reducing test coverage; the existing test ensured the `from_network` initializer worked properly.  Please restore.

Reply:
right but existing test was based on the assumption that someone can pass a bogus string for network name which is not the case anymore. `validate_contracts_loaded` should be enough for testing this method. `from_network` is deprecated and will be removed in upcoming versions

Reply:
I hadn't anticipated deprecating `from_network`.  It's useful to ensure the node is serving the expected network (would produce _contract not found_ error otherwise) and to use our dockerized testnet.  I suppose users could still use `from_json` for this purpose though.